examples/pipeline/sample.tfvars

```diff
@@ -28,6 +28,7 @@ organization = {
   disable_sso_management      = false
   deploy_audit_role           = true
   audit_role_name             = "security-audit"
+  audit_role_template_url     = "https://s3.amazonaws.com/pht-cloudformation/aws-account-automation/AuditRole-Template.yaml"
```

Comment:
Even though it's not required, I figured it would still be a good idea to highlight that it's configurable in the example here; I'd be good removing it and using the default if you prefer.